src/transaction/log_page_buffer.c

```diff
@@ -7676,14 +7677,27 @@ logpb_daemons_destroy ()
 
 #if defined(SERVER_MODE)
 /*
- * logpb_do_checkpoint -
- *
- * return:
- *
- * NOTE:
+ * logpb_wakeup_remove_log_archive_daemon () - wake up remove log archive daemon
+ */
+void
+logpb_wakeup_remove_log_archive_daemon ()
+{
+  if (prm_get_integer_value (PRM_ID_REMOVE_LOG_ARCHIVES_INTERVAL) == 0)
```

Comment:
I know this is rather right from the point of the param, however, I'd like to propose keeping the existing behavior. 
A log archive is purged at every PRM_ID_REMOVE_LOG_ARCHIVES_INTERVAL seconds or the moment of archiving.